Source/Common/Preferences.cpp

```diff
@@ -857,10 +857,10 @@ int Preferences::ExplorerShell()
                 }
             }
         }
-        ExplorerShell_Edit("SystemFileAssociations\\audio", 0, IsChanged);
-        ExplorerShell_Edit("SystemFileAssociations\\Directory.Audio", 0, IsChanged);
-        ExplorerShell_Edit("SystemFileAssociations\\Directory.Video", 0, IsChanged);
-        ExplorerShell_Edit("SystemFileAssociations\\video", 0, IsChanged);

```

Comment:
Shouldn't we delete the MediaInfo related entries in case it is from a previous version?

Reply:
The previous code (now in red) is trying to remove entries from `HKCU\SystemFileAssociations` if I understand correctly? See https://github.com/MediaArea/MediaInfo/pull/960#issuecomment-2482925278. Since that key does not exist, nothing happens. That is why it fails to completely disable the shell ext previously. Now it is changed to delete the correct key but actually may not be necessary as the key is now deleted by NSIS on upgrade/install. I am not sure if the key exists on portable. The one I have was created by NSIS.

Reply:
My mistake. 

>  I am not sure if the key exists on portable. The one I have was created by NSIS.

Portable is the cpp file you are checking without the NSIS, so no diff.